tool/site/server/src/services/environment.ts

```diff
@@ -1,9 +1,17 @@
-import {CONFIG} from "../config"
+export type Environment = "internal-dev" | "internal-dev-sandbox" | "internal-qa" | "int"
 
-export function isLocal(): boolean {
-  return CONFIG.environment.endsWith("-sandbox") ?? false
+export const internalDev = "internal-dev"
+export const internalDevSandbox = "internal-dev-sandbox"
+export const int = "int"
```

Comment:
yeah I think this might be easier to manage as an enum

Reply:
same as above comment